azure-pipelines.yml

```diff
@@ -151,24 +145,6 @@ stages:
               publisherId: "$(publisher)"
               extensionId: "sonarcloud-test"
               accounts: "https://devops.azure.com/$(itOrg)"
-          - task: Bash@3
-            displayName: "Wait for tasks to be available in CI"
-            env:
-              SERVICE_URL: "https://devops.azure.com/$(SonarSourceITOrganization)"
-              AZURE_TOKEN: $(SonarSourceITOrganizationPAT)
-            inputs:
-              targetType: "inline"
-              script: |
-                extension_installed=""
-                while [ -z "$extension_installed" ]; do
-                  echo "Waiting for SonarCloudPublishTest task to be registered..."
-                  tasks=$(tfx build tasks list --authType pat --service-url "$SERVICE_URL" -t "$AZURE_TOKEN" --no-color --json)
-                  extension_installed=$(echo $tasks | jq -r ".[] | select(.name == \"SonarCloudPublishTest\") | .id")
-                  if [ -z "$extension_installed" ]; then
-                    sleep 5
-                  fi
-                done
-                echo "SonarCloudPublishTest task is registered with ID: $extension_installed"
```

Comment:
I reckon that now that we're not uninstalling the extension anymore, that's not needed 🤔

Reply:
exactly, because the tasks are always there; idk if there's a way to wait for the new version of a task to be extra sure